website/docs/workflows/reference/workflow-triggers.md

```diff
@@ -0,0 +1,82 @@
+
+## Scheduled trigger
+
+A scheduled trigger starts a workflow at specific times or regular intervals, making it useful for automating tasks that need to run on a fixed schedule. For example, you can set up a scheduled trigger to automatically send a daily report summarizing sales performance to the sales team. 
+
+### Timezone
+
+Select the appropriate timezone from the dropdown menu. The default value is the system's timezone, such as `UTC+05:30 - Asia/Kolkata`. Ensure that the selected timezone aligns with when the workflow runs.
+
+### Set schedule
+
+The workflow schedule uses a standard 5-field cron expression to define when the workflow runs. To schedule a workflow, configure the fields—**Minute**, **Hour**, **Day of Month**, **Month**, and **Day of Week**.
+
+You can use the following syntax in each field to configure the schedule:
+
+- **Asterisk (`*`)**: Allows you to include a wildcard that represents all possible values in a field. For example, when set for all fields, the expression (`* * * * *`) means the workflow runs every minute of every hour, on every day, in every month, and on every day of the week.
+
+- **Comma (`,`)**: Allows you to set more than one value in a field. For example, when set to `0,30` in **Minute** and an asterisk for the others, the expression (`0,30 * * * *`) means the workflow runs at minute `0` and `30` of every hour.
+
+- **Hyphen (`-`)**: Allows you to define a range of values in a field. For example, when set to `0` in **Minute**, `1-5` in **Hour**, and an asterisk for the others, the expression (`0 1-5 * * *`) means the workflow runs every hour from `1 AM` to `5 AM`.
+
+- **Slash (`/`)**: Used to specify intervals for a field. For example, when set to `*/5` in **Minute** and an asterisk for the others, the expression (`*/5 * * * *`) means the workflow runs every 5 minutes, starting from minute `0` and running at 5, 10, 15, and so on.
+
+#### Popular cron expressions
+
+| Schedule  | Cron expression  | Minute | Hour | Day of Month | Month | Day of Week |
+|-----------|------------------|--------|------|--------------|-------|-------------|
+| Every minute | `* * * * *`      | `*`    | `*`  | `*`          | `*`   | `*`         |
+| Every hour at minute 0  | `0 * * * *`      | `0`    | `*`  | `*`          | `*`   | `*`         |
+| Every day at noon | `0 12 * * *`     | `0`    | `12` | `*`          | `*`   | `*`         |
+| Every Sunday at midnight | `0 0 * * 0`      | `0`    | `0`  | `*`          | `*`   | `0`         |
+
+For more help with generating cron expressions, you can use the [Appsmith Cron App](https://app.appsmith.com/app/cron/home-66fac6586931e9259851a137) or online tools like [Crontab Guru](https://crontab.guru/) and [CronMaker](http://www.cronmaker.com/).
```

Comment:
Just link our app only, other links have paths to their own services which I don't want to promote in our docs.